lib/livebook/hubs/team_client.ex

```diff
@@ -589,6 +587,25 @@ defmodule Livebook.Hubs.TeamClient do
     )
   end
 
+  defp handle_event(:app_deployment_deleted, %Teams.AppDeployment{} = app_deployment, state) do
+    deployment_group_id = app_deployment.deployment_group_id
+
+    with {:ok, deployment_group} <- fetch_deployment_group(deployment_group_id, state) do
+      if deployment_group.id == state.deployment_group_id do
+        manager_sync()
+      end
+    end
+
+    Teams.Broadcasts.app_deployment_deleted(app_deployment)
+    remove_app_deployment(state, app_deployment)
```

Comment:
Just so I get it right, the idea is that the deployment group only contains running app deployments, that's why we remove from the list? And on subsequent connection, the app deployment is not going to be in the list either?

Reply:
Correct!

Reply:
@aleDsz I would still call it `app_deployment_stopped` to align the wording with Teams and make it easier to follow (could also be revoke). But if you have a strong preference, it's fine!

Reply:
If we want to call it `_stopped`, then maybe we should have the other as `_started`.

Reply:
Works for me! One other idea would be to have "active" app deployments and `_activated` and `_deactivated`, if we prefer a distinct word.

Reply:
I would +1 for "active" idea, because would match with Teams